docs/conf.py

```diff
@@ -178,6 +178,13 @@
 # Output file base name for HTML help builder.
 htmlhelp_basename = 'togadoc'
 
+try:
```

Comment:
I'm interested in your reasoning for adding this. Does it look nicer? Will it 'just work' on readthedocs? Can you show a before/after, or something similar?